documentation/docs/98-reference/20-$app-types.md

```diff
@@ -0,0 +1,91 @@
+---
+title: $app/types
+---
+
+This module contains generated types for the routes in your app.
+
+<blockquote class="since note">
+	<p>Available since 2.26</p>
+</blockquote>
+
+```js
+// @noErrors
+import type { RouteId, RouteParams, LayoutParams } from '$app/types';
+```
+
+## Asset
+
+A union of all the filenames of assets contained in your `static` directory.
+
+<div class="ts-block">
+
+```dts
+type Asset = '/favicon.png' | '/robots.txt';
+```
+
+</div>
+
+## RouteId
+
+A union of all the route IDs in your app. Used for `page.route.id` and `event.route.id`.
+
+<div class="ts-block">
+
+```dts
+type RouteId = '/' | '/my-route' | '/my-other-route/[param]';
+```
+
+</div>
+
+## Pathname
+
+A union of all valid pathnames in your app.
+
+<div class="ts-block">
+
+```dts
+type Pathname = '/' | '/my-route' | `/my-other-route/${string}` & {};
+```
+
+</div>
+
+## ResolvedPathname
+
+`Pathname`, but possibly prefixed with a [base path](https://svelte.dev/docs/kit/configuration#paths). Used for `page.url.pathname`.
+
+<div class="ts-block">
+
+```dts
+type Pathname = `${'' | `/${string}`}/` | `${'' | `/${string}`}/my-route` | `${'' | `/${string}`}/my-other-route/${string}` | {};
```

Comment:
there is a small typo here
```suggestion
type ResolvedPathname = `${'' | `/${string}`}/` | `${'' | `/${string}`}/my-route` | `${'' | `/${string}`}/my-other-route/${string}` | {};
```

Reply:
Hi there. Can you create a new PR to fix this or an issue track it? Commenting on closed PRs is generally hard to keep track of